apis/flowcollector/v1beta1/flowcollector_types.go

```diff
@@ -910,7 +911,8 @@ type DebugConfig struct {
 // `SubnetLabels` allows to define custom labels on subnets and IPs or to enable automatic labelling of recognized subnets in OpenShift.
 type SubnetLabels struct {
 	// `openShiftAutoDetect` allows, when set to `true`, to detect automatically the machines, pods and services subnets based on the
-	// OpenShift install configuration and the Cluster Network Operator configuration.
+	// OpenShift install configuration and the Cluster Network Operator configuration. Indirectly, this is a way to accurately detect
+	// external traffic: flows that are not labeled for those subnets are external to the cluster. Enabled by default on OpenShift.
```

Comment:
do u want to add
`	//+kubebuilder:default:=true`

Reply:
I prefer to keep nil by default , and have default handling in code, it's more flexible and allows us to change the default values between releases